CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 - (Bugfix) Ensure Wait actions to be present after AddMember
 - (Documentation) Refactor metrics (Part 1)
 - (Bugfix) Extend Agency HealthCheck for replace
+- (Refactor) Use cached member's clients 
```

Comment:
is it refactoring or feature (as stated in PR title)?

Reply:
refactoring